homeassistant/components/zha/config_flow.py

```diff
@@ -164,9 +164,11 @@ async def async_step_zeroconf(
         """Handle zeroconf discovery."""
         # Hostname is format: livingroom.local.
         local_name = discovery_info.hostname[:-1]
+        radio_type = discovery_info.properties.get("radio_type") or local_name
         node_name = local_name[: -len(".local")]
         host = discovery_info.host
-        device_path = f"socket://{host}:6638"
+        port = discovery_info.port
```

Comment:
```suggestion
        port = 6638 if local_name.startswith("tube") else discovery_info.port
```

Looks like we are stuck with the hard coding